scripts/schema.sql

```diff
@@ -190,4 +190,32 @@ CREATE TABLE IF NOT EXISTS `achievements_progress` (
     PRIMARY KEY (`achievement_id`, `user_id`),
     FOREIGN KEY (`achievement_id`) REFERENCES `list_of_achievements`(`achievement_id`),
     FOREIGN KEY (`user_id`) REFERENCES `users`(`user_id`)
-)
\ No newline at end of file
+);
```

Comment:
Bump schema version in line 5 and in `db.py`. Trying to keep track of version changes with our db when we need to refer to past data.

Reply:
Fixed